spec/models/user_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+
+  it 'is valid with first name, last name, e-mail and password' do
+    user = User.new(
```

Comment:
generate test data with factorybot.
also read about let() and let()! and use it, not local variables for storing objects in tests.